# MLe-KCNQ2
-Here, we provide the Supplementary Material of the work "Improved KCNQ2 gene missense variant interpretation with artificial intelligence"

### ABSTRACT
Advances in DNA sequencing technologies have revolutionised rare disease diagnosis, resulting in an increasing volume of available genomic data. Despite this wealth of information and improved procedures to combine data from various sources, identifying the pathogenic causal variants and distinguishing between severe and benign variants remains a key challenge. Mutations in the Kv7.2 voltage-gated potassium channel gene (KCNQ2) have been linked to different subtypes of epilepsies, such as benign familial neonatal epilepsy (BFNE) and epileptic encephalopathy (EE). To date, there is a wide variety of genome-wide computational tools aiming at predicting the pathogenicity of variants. However, previous reports suggest that these genome-wide tools have limited applicability to the KCNQ2 gene related diseases due to overestimation of deleterious mutations and failure to correctly identify benign variants, being, therefore, of limited use in clinical practice. In this work, we found that combining readily available features, such as AlphaFold structural information, Missense Tolerance Ratio (MTR) and other commonly used protein descriptors, provides foundations to build reliable gene-specific machine learning ensemble models. Here, we present a transferable methodology able to accurately predict the pathogenicity of KCNQ2 missense variants with unprecedented sensitivity and specificity scores above 90%.

